Library/Homebrew/cask/lib/hbc/artifact/binary.rb

```diff
@@ -9,7 +9,7 @@ def install_phase
 
       def link
         super
-        FileUtils.chmod "+x", source
+        @command.run!("/bin/chmod", args: ["+x", source])
```

Comment:
Looks good, just need to fix the indentation on the last few lines (or run `brew style --fix`).